test/src/imgur/getTags.test.ts

```diff
@@ -1,8 +1,13 @@
+import { ImgurClient } from 'imgur'
+
 const imgurGetTagsMethod = 'biketag.images.getTags'
+
 describe(imgurGetTagsMethod, () => {
   test.todo(`${imgurGetTagsMethod} method requires ImgurHash from payload`)
 
+  // Validate `data` field for all three branches of payload conditional
   test.todo(`${imgurGetTagsMethod} method resolves data of type Tag[]`)
 
+  // Check `getTags` result.status === HttpStatusCode.Ok
   test.todo(`${imgurGetTagsMethod} method resolves status of HttpStatusCode.Ok`)
```

Comment:
💡 Since we're checking the results of all three conditional branches, it might also make sense to drop this and roll httpstatus check into each of those.

Reply:
@nonara Yes, that is fine. This HttpStatusCode.Ok was mostly a placeholder and I think each of the status messages needs to be refactored. This is a scenario in which the test writing will inform the code.